### DISCUSSION TITLE
Is there a way to hide "success" log lines in "gatsby build" output?

### DISCUSSION THREAD

**Discussion by dko-slapdash** [gatsbyjs/gatsby#27820] [Ideas / Feature Requests]
Gatsby build output is pretty verbose: it prints lots of "success" or "indo" entries, like these:

![image](https://user-images.githubusercontent.com/46383452/98058195-f38b0000-1df8-11eb-8fbb-600cf472a04f.png)

They pollute CI logs a bit.

Is there a way to show only warnings/errors and minimize the rest of the output? I couldn't find the corresponding option (or an environment variable) in the docs.

**Comment by ascorbic** [comment]
Hey. There's not a specific way to hide all output, but it might be a useful addition, so I'll tag this as a feature request. In the meantime, the simplest way is to filter the output:

```shell
gatsby build | grep -v success
```

**Reply by dko-slapdash** [reply]
Thanks. It’s not as simple unfortunately, because this command will always return zero as exit code and not the exit code of gatsby. There must be a bashism for returning the exit code of gatsby too though, like

set -opipefail; gatsby build | (grep -v success || true)
